lib/assessment.md

```diff
@@ -2,53 +2,71 @@ This file is a **markdown** file. Markdown is a special way of formatting text,
 
 In responding to some of these questions, you may be asked to write HTML or CSS code. Please do so using the following Markdown formatting.
 
-##### For HTML
-    ```HTML
-    <html>
-    ...
-    </html>
-    ```
-
-##### For CSS
-    ```CSS
-    ul {
-      ...
-    }
-    ```
+# For HTML
 
-Using this format will make it easier for us to grade you, so that we can get everyone their feedback faster. Thanks for your help!
+````
+```HTML
+<html>
+...
+</html>
+```
+````
 
-## Question 1
+# For CSS
 
-When we say that the web is a service, what does that mean? Explain the interaction in terms of clients and servers.
-<!-- your answer starts here -->
+````
+```CSS
+ul {
+  ...
+}
+```
+````
 
-<!-- your answer ends here -->
+Using this format will make it easier for us to grade you, so that we can get everyone their feedback faster. Thanks for your help!
+
+## Question 1
+When we say that the web is a service, what does that mean? Explain the interaction in terms of clients and servers. <!-- your answer starts here --> The web is a service in that it represents a consistent framework for displaying documents across the internet <!-- your answer ends here -->
 
 ## Question 2
+Between the comments below, write HTML to create a basic web page (using the format described in the instructions). The page should have a title in the head and an unordered list (with three items) inside the body. <!-- your answer starts here -->
 
-Between the comments below, write HTML to create a basic web page (using the format described in the instructions). The page should have a title in the head and an unordered list (with three items) inside the body.
-<!-- your answer starts here -->
+<!DOCTYPE html>
+<html>
+<head>
+<title>Meerkat Central</title>
+<head/>
+<body>
+<ul>
+<li>Meerkat?</li>
+<li>Meerkat.</li>
+<li>Meer. Kat.</li>
+</ul>
+</body>
+</html>
```

Comment:
Your content is correct here, but you really need to fix that indenting - it's important for readability.